### PR TITLE
ECHO-757: updated account operations

### PR DESCRIPTION
**Pull request by dvolynets**

### CODE REVIEW

File: api-reference/echo-operations/account-management.md [hunk 1]
```diff
@@ -7,39 +7,28 @@ This operation is used to create an account
...
-
-  struct fee_parameters_type
-  {
-      uint64_t basic_fee      = 5*ECHO_BLOCKCHAIN_PRECISION; ///< the cost to register the cheapest non-free account
-      uint64_t premium_fee    = 2000*ECHO_BLOCKCHAIN_PRECISION; ///< the cost to register the cheapest non-free account
-      uint32_t price_per_kbyte = ECHO_BLOCKCHAIN_PRECISION;
-  };
-
-  asset           fee;
-  /// This account pays the fee. Must be a lifetime member.
```
Comment (aliakseikazak) [inline api-reference/echo-operations/account-management.md:26]: `/// This account pays the fee. Must be a lifetime member.`
Delete row - any account can create another account.

File: api-reference/echo-operations/account-management.md [hunk 2]
```diff
@@ -105,33 +79,31 @@ This operation is used to update an existing account. It can be used to update t
...
+   account_id_type account;
+   /// New active authority. This can be updated by the current active authority.
+   optional<authority> active;
+   // New ED25519 public key
+   optional<eddsa::public_key_t> echorand_key;
+   /// New account options
+   optional<account_options> new_options;
+   
+   /// New account options
+   optional<account_options> new_options;
```
Comment (aliakseikazak) [inline api-reference/echo-operations/account-management.md:103]: `   /// New account options
   optional<account_options> new_options;`
duplicate

File: api-reference/echo-operations/account-management.md [hunk 3]
```diff
@@ -147,40 +119,14 @@ struct account_update_operation : public base_operation
...
-      /// will be counted as voting for the referenced account's selected votes instead.
-      "voting_account": "1.2.5",
-      "num_witness": 0,   /// The number of active witnesses this account votes the blockchain should appoint
-      "num_committee": 0, /// The number of active committee members this account votes the blockchain should appoint
-      "votes": [],        /// This is the list of vote IDs this account votes for.
-      "extensions": []
-    },
     "extensions": {}
   }
 ]
```
Comment (aliakseikazak) [inline api-reference/echo-operations/account-management.md]: **improve:** add more informative example, for example:
```
{
    "fee": {
        "amount": 0,
        "asset_id": "1.3.0"
    },
    "account": "1.2.0",
    "new_options": {
        "voting_account": "1.2.5",
        "delegating_account": "1.2.6",
        "num_committee": 0,
        "votes": [],
        "extensions": []
    },
    "extensions": []
}
```

File: api-reference/echo-operations/account-management.md (hunk 2 above)
Comment (dvolynets) [reply api-reference/echo-operations/account-management.md:103]: fixed

File: api-reference/echo-operations/account-management.md (hunk 3 above)
Comment (dvolynets) [reply api-reference/echo-operations/account-management.md]: fixed

File: api-reference/echo-operations/account-management.md (hunk 1 above)
Comment (dvolynets) [reply api-reference/echo-operations/account-management.md:26]: fixed

**Review by aliakseikazak — CHANGES_REQUESTED** [review]
left a couple of comments

**Review by aliakseikazak — APPROVED** [review]

**Pull request merged**